OpenAI/Packages/com.openai.unity/Runtime/Models/Model.cs

```diff
@@ -16,9 +16,15 @@ public sealed class Model
         /// Constructor.
         /// </summary>
         /// <param name="id"></param>
-        public Model(string id)
+        public Model(string id, string ownedBy = null)
         {
+            if (string.IsNullOrWhiteSpace(id))
+            { 
+                throw new ArgumentNullException(nameof(id), "Is null, empty, or whitespace");
```

Comment:
```suggestion
                throw new ArgumentNullException(nameof(id), "Missing the id of the specified model.");
```

Reply:
Do you want periods at the end of these comments, or no periods.  It seems to be hit or miss.